image/ovn/Dockerfile

```diff
@@ -20,11 +22,19 @@ COPY run_ovsdb_etcd.sh /run_ovsdb_etcd.sh
 COPY run_ovsdb_etcd_sb.sh /run_ovsdb_etcd_sb.sh
 COPY run_ovsdb_etcd_nb.sh /run_ovsdb_etcd_nb.sh
 
-RUN /install_ovn.sh $USE_OVN_RPMS $EXTRA_OPTIMIZE
+RUN /install_ovn.sh $USE_OVN_RPMS $USE_OVN_DEBS $EXTRA_OPTIMIZE
 RUN /install_utils_from_sources.sh $INSTALL_UTILS_FROM_SOURCES
 RUN /install_ovsdb_etcd.sh
 RUN /install_etcd.sh
 
+RUN if [ "$USE_OVN_DEBS" = "yes" ]; then \
```

Comment:
Why do we need to do this only when installing deb packages?  Doesn't this make sense for both rpm-based and deb-based distros?

Reply:
Hey @dceara makes sense. I did the fix.

Reply:
Sorry, I broke something I will fix in the next commit.

Reply:
hey @dceara could you follow with the review? Thanks